python/pyspark/ml/classification.py

```diff
@@ -1063,6 +1064,13 @@ class MultilayerPerceptronClassifier(JavaEstimator, HasFeaturesCol, HasLabelCol,
     True
     >>> model.weights == model2.weights
     True
+    >>> mlp2 = mlp2.setInitialWeights([2, 5, 1, -7, -5, -10, 0, 0.6, -1, 2, -2, 1, 2, -7, -1, -2,
+    ...    2, 1, -1, 9, -9, 3, -3, -3, 3.0, 0, -1])
```

Comment:
In MLlib, Scala unit tests focus on verifying the correctness, Java unit tests focus on compatibility verification, and Python doc tests should focus on providing examples to users. So I think we should keep Python doc tests succinct just as you see in the code base.